src/planwise/component/datasets.clj

```diff
@@ -42,6 +45,12 @@
   (->> (select-datasets-for-user (get-db store) {:user-id user-id})
        (map db->dataset)))
 
+(defn list-datasets-with-import-jobs
+  [store]
+  (->> (select-datasets-with-import-jobs (get-db store))
+       (map db->dataset)
+       (filter (comp some? :import-job))))
```

Comment:
It's a leftover from a bug that caused finished jobs to be stored as `nil`, so it should not be necessary. Still, it is a safeguard; as such, I'd leave it there, though it should not be needed.